grantnav/frontend/templates/search.html

```diff
@@ -30,8 +30,20 @@
           </div>
         </div>
       </form>
+
+      {% if advance_search_info %}
```

Comment:
It's 'advanced search' in all of these cases, not 'advance search'

Reply:
ups yes! Thanks! I have now fixed it.